includes/admin/class-wc-rest-payments-deposits-controller.php

```diff
@@ -102,6 +112,13 @@ public function get_deposits_overview() {
 		return $this->forward_request( 'get_deposits_overview', [] );
 	}
 
+	/**
+	 * Retrieve an overview of all deposits from the API.
+	 */
+	public function get_all_deposits_overviews() {
+		return $this->forward_request( 'get_all_deposits_overviews', [] );
```

Comment:
__A note about testing:__

The best environment to test this in would be one with a WCPay account, which has multiple bank accounts with different currencies associated with it, as well as completed payouts to those accounts.

In my case I have a fresh setup with multiple bank accounts, within a fresh Stripe Connect environment, and I have to wait seven days for the payouts to happen, which I can obviously not afford. To test the UI, add this line:

```suggestion
		echo file_get_contents( WCPAY_ABSPATH . '/all-deposits-overview.json' ); exit;
		return $this->forward_request( 'get_all_deposits_overviews', [] );
```

And this file to the root of the plugin:

```json
{
    "deposit": {
        "last_paid": [
          {
            "id": "po_...",
            "date": 1619395200000,
            "type": "deposit",
            "amount": 3160,
            "status": "paid",
            "bankAccount": " ••••  (USD)",
            "currency": "eur",
            "automatic": true,
            "fee": 0,
            "fee_percentage": 0,
            "created": 1619395200
          },
          {
            "id": "po_...",
            "date": 1619395200000,
            "type": "deposit",
            "amount": 1000,
            "status": "paid",
            "bankAccount": " ••••  (EUR)",
            "currency": "gbp",
            "automatic": true,
            "fee": 0,
            "fee_percentage": 0,
            "created": 1619395200
          }
        ],
        "next_scheduled": [
            {
                "id": "wcpay_estimated_weekly_eur_1622678400",
                "date": 1622678400000,
                "type": "deposit",
                "amount": 3343,
                "status": "estimated",
                "bankAccount": null,
                "currency": "eur",
                "automatic": true,
                "fee": 0,
                "fee_percentage": 0,
                "created": 1622678400
            },
            {
                "id": "wcpay_estimated_weekly_gbp_1622678400",
                "date": 1622678400000,
                "type": "deposit",
                "amount": 1656,
                "status": "estimated",
                "bankAccount": null,
                "currency": "gbp",
                "automatic": true,
                "fee": 0,
                "fee_percentage": 0,
                "created": 1622678400
            }
        ]
    },
    "balance": {
        "pending": [
            {
                "amount": 3343,
                "currency": "eur",
                "source_types": {
                    "card": 3343
                }
            },
            {
                "amount": 1656,
                "currency": "gbp",
                "deposits_count": 2,
                "source_types": {
                    "card": 1656
                }
            }
        ],
        "available": [
            {
                "amount": 2030,
                "currency": "eur",
                "source_types": {
                    "card": 0
                }
            },
            {
                "amount": 0,
                "currency": "gbp",
                "source_types": {
                    "card": 0
                }
            }
        ]
    },
    "account": {
        "deposits_enabled": true,
        "deposits_schedule": {
            "delay_days": 7,
            "interval": "weekly",
            "weekly_anchor": "thursday"
        },
        "default_currency": "gbp"
    }
}
```